Merge 19.9.0.1 beta

mokagio opened the pull request:
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

---

Includes:

- [Fix crash due to an unregistered cell in Stats Insights](https://github.com/wordpress-mobile/WordPress-iOS/pull/18613) by @Gio2018 
- [(Internal change) Fix default_tab_experiment prop. for account_created event](https://github.com/wordpress-mobile/WordPress-iOS/pull/18625) by @leandroalonso 


Nothing to test, if CI Shows green, we're good.

Commit:
Update SiteStatsInsightsTableViewController, register ViewsVisitorsRow to prevent a crash due to unregistered cell

Commit:
Update release notes for PR #18613

sla8c committed:
remove FeatureFlag from InsightsTableViewController

sla8c committed:
viewsVisitors moved to end

leandroalonso committed:
fix: fill the correct default_tab property for created account

mokagio committed:
Merge branch 'release/19.9' into issue/crash-for-unregistered-cell

mokagio committed:
Fix `default_tab_experiment` prop. for `account_created` event (#18625)

mokagio committed:
Fix crash due to an unregistered cell in Stats Insights (#18613)

mokagio committed:
Update app translations – `Localizable.strings`

mokagio committed:
Update WordPress metadata translations

mokagio committed:
Update Jetpack metadata translations

mokagio committed:
Bump version number

mokagio commented on fastlane/metadata/es-MX/description.txt:
```diff
@@ -0,0 +1,12 @@
+Crea y gestiona tu web o blog WordPress directamente desde tu dispositivo iOS: crea y edita cualquier entrada o página, sube tus fotos y videos favoritos, accede a las estadísticas o responde a los comentarios fácilmente.
+ 
+Con WordPress para iOS, podrás publicar todo tu contenido desde la palma de tu mano. Escribe desde tu sofá el borrador de ese haiku que no te quitas de la cabeza. Sube y comparte en un momento esa increíble foto que hiciste durante el descanso en el trabajo. Responde a los últimos comentarios que te hayan dejado o descubre a qué nuevos países ha llegado hoy tu contenido echándole un vistazo a la  ...
+ 
+WordPress para iOS es un proyecto de código abierto, lo que significa que tú también puedes participar en su desarrollo. Descubre más información en https://apps.wordpress.com/contribute/.
+ 
+WordPress para iOS es compatible con sitios de WordPress.com y sitios de WordPress.org autoalojados que tengan WordPress 4.0 o superior.
+
+¿Necesitas ayuda con la aplicación? Entra en el foro de ayuda https://ios.forums.wordpress.org/ o déjanos un tweet en @WordPressiOS.
+
+Mira el aviso de privacidad para usuarios de California en https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.
+
```
This and the other `es-MX` files got deleted [because of a failed download during the 19.8 finalization](https://github.com/wordpress-mobile/WordPress-iOS/pull/18612/). This time around, the download was successful so here they are again.

AliSoftware replied:
🎉

mokagio committed:
Update draft release notes for 19.9.

mokagio committed:
Update draft release notes for 19.9.

mokagio committed:
Release Notes: add new section for next version (20.0)

mokagio committed:
Fix double `20.0` header in `RELEASE-NOTES.txt`

That was due to me running the automation to update the release notes.

AliSoftware commented on fastlane/metadata/default/release_notes.txt:
```diff
@@ -1,14 +1,10 @@
-You can now upload videos of any length to self-hosted sites—the only limit is your own imagination. (Oh, and the 2 GB file size limit, we guess.)
+* [*] Site Settings: we fixed an issue that prevented the site title to be updated when it changed in Site Settings [#18543]
+* [*] Media Picker: Fixed an issue where the empty state view was being displayed incorrectly. [#18471]
+* [*] Quick Start: We are now showing a different set of Quick Start tasks for existing sites and new sites. The existing sites checklist includes new tours such as: "Check your notifications" and "Upload photos or videos".  [#18395, #18412, #18443, #18471]
+* [*] Site Creation: we fixed an issue where the navigation buttons were not scaling when large fonts were selected on the device [#18559]
+* [**] Block Editor: Cover Block: Improve color contrast between background and text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4808]
+* [***] Block Editor: Add drag & drop blocks feature [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4832]
+* [*] Block Editor: Gallery block: Fix broken "Link To" settings and add "Image Size" settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4841]
+* [*] Block Editor: Unsupported Block Editor: Prevent WordPress.com tour banner from displaying. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4820]
+* [*] Widgets: we fixed an issue where text appeared flipped in rtl languages [#18567]
```
I'm having a brain fart here as to what updates this file / when is it updated by our scripts. Is it during the download of metadata from GlotPress that we copy the value [from `WordPress/Resources/release_notes.txt`](https://github.com/wordpress-mobile/WordPress-iOS/pull/18631/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bc) into this file? Or is it only done one-time during code freeze and not updated later if the `WordPress/Resources/release_notes.txt` change like here in this PR?

In any case, feels like it wouldn't hurt updating those `fastlane/metadata/default` draft notes with the 2 new items [you just added to `WordPress/Resources/release_notes.txt`](https://github.com/wordpress-mobile/WordPress-iOS/pull/18631/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bc)? 🙃

mokagio replied:
> Is it during the download of metadata from GlotPress that we copy the value [from WordPress/Resources/release_notes.txt](https://github.com/wordpress-mobile/WordPress-iOS/pull/18631/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bc) into this file?

Correct, that's the only time it happens.

> In any case, feels like it wouldn't hurt updating those fastlane/metadata/default draft notes with the 2 new items [you just added to WordPress/Resources/release_notes.txt](https://github.com/wordpress-mobile/WordPress-iOS/pull/18631/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bc)? 🙃

Fair point.

mokagio replied:
Done: [`79180f1`](https://github.com/wordpress-mobile/WordPress-iOS/pull/18631/commits/79180f129f9663a50dbb11db56cd22024142d2f7)

AliSoftware commented on fastlane/jetpack_metadata/default/release_notes.txt:
```diff
@@ -1,12 +1,10 @@
-When you Follow Conversation in Reader post cards and post details, in-app notifications are now opt-out rather than opt-in.
+* [*] Site Settings: we fixed an issue that prevented the site title to be updated when it changed in Site Settings [#18543]
+* [*] Media Picker: Fixed an issue where the empty state view was being displayed incorrectly. [#18471]
+* [*] Quick Start: We are now showing a different set of Quick Start tasks for existing sites and new sites. The existing sites checklist includes new tours such as: "Check your notifications" and "Upload photos or videos".  [#18395, #18412, #18443, #18471]
+* [*] Site Creation: we fixed an issue where the navigation buttons were not scaling when large fonts were selected on the device [#18559]
+* [**] Block Editor: Cover Block: Improve color contrast between background and text [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4808]
+* [***] Block Editor: Add drag & drop blocks feature [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4832]
+* [*] Block Editor: Gallery block: Fix broken "Link To" settings and add "Image Size" settings [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4841]
+* [*] Block Editor: Unsupported Block Editor: Prevent WordPress.com tour banner from displaying. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4820]
+* [*] Widgets: we fixed an issue where text appeared flipped in rtl languages [#18567]
```
Same remark/question as for `fastlane/metadata/default/release_notes.txt` vs `Wordpress/Resources/release_notes.txt`

AliSoftware reviewed (approved):
- [x] Changes from bugfix PRs which landed in the release branch
- [x] Version bump in `.xcconfig` files
- [x] Latest app (`Localizable.strings`) and metadata (`fastlane/*metadata/**/*.txt`) translations pulled from Glotpress

Left a question about the `fastlane/metadata/default` release notes not matching the ones in `WordPress/Resources/` but I'm pretty sure that's not a blocker nor a big deal because the ones that Grace uses for the wordsmithing are the `WordPress/{Jetpack/}Resources` ones and the files will be updated later once we get the editorialized version before being sent for translation anyway, so shouldn't have any impact.

mokagio committed:
Add latest entries to Fastlane `release_notes.txt`, too

See
https://github.com/wordpress-mobile/WordPress-iOS/pull/18631#discussion_r874644807
